### DISCUSSION TITLE
Email verification issues in Reverify your academic affiliation

### DISCUSSION THREAD

**Discussion by kulorty** [community/community#66034] [GitHub Education]
### Hi everyone,

I'm having trouble with the email verification part when I reapply for the GitHub student pack. The system requires a school-issued email address, but the email address I requested from my school was "@stu.xxx.edu.com" instead of "@xxx.edu.com", which led to the system incorrectly assuming that I wasn't using a school-issued email address. e-mail address. Also, I am unable to redo the application, what should I do.

**Comment by arafajs** [comment]
In this case, I recommend that you contact GitHub support for assistance.
Go to the GitHub Support website: https://github.com/contact.
